index.html

```diff
@@ -49,10 +73,208 @@ <h2 class="CONNECT"><a href="#">LET'S CONNECT</a> </h2>
+              <div >
+                  <ul class="languages">
+                  <li class="language">HTML</li>
+                  <li class="language">CSS</li>
+                  <li class="language">javascript</li>
+                  </ul>
+              </div>
+              <div><button class="see-project" type="button">See project</button></div>
+          </div>
+            </div>
+          
+      
+      
+      
+            <div class="Works">
+              <div class="img-1"><img class="fontimg"  width="100%"  height="100%" src="Snapshoot2.png" alt=""></div>
+      
+      
+              <div class="tonic">
+                  <div><h1>Muti-Post <br>Stories</h1></div>
+                  <div class="flex1">
+                      <div>Caponi</div>
+                      <div class="round"></div>
+                      <div>back End</div>
+                      <div class="round"></div>
+                      <div>2015</div>
+                  </div>
+                  <div>
+                      <p>
+                          A dayly selection of privatelypersonalized reads:no accounts <br>
+                          or sign-ups required.
+                      </p>
+                  </div>
+                  <div >
+                      <ul class="languages">
+                      <li class="language">HTML</li>
+                      <li class="language">CSS</li>
+                      <li class="language">javascript</li>
+                      </ul>
+                  </div>
+                  <div><button class="see-project" type="button">See project</button></div>
+              </div>
+                </div>
+           
+          
+          
+          
+                <div class="Works">
+                  <div class="img-1"><img class="fontimg"  width="100%"  height="100%" src="Snapshoot3.png" alt=""></div>
+          
+          
+                  <div class="tonic">
+                      <div><h1>Tonic</h1></div>
+                      <div class="flex1">
+                          <div>Caponi</div>
+                          <div class="round"></div>
+                          <div>back End</div>
+                          <div class="round"></div>
+                          <div>2015</div>
+                      </div>
+                      <div>
+                          <p>
+                              A dayly selection of privatelypersonalized reads:no accounts <br>
+                              or sign-ups required.
+                          </p>
+                      </div>
+                      <div >
+                          <ul class="languages">
+                          <li class="language">HTML</li>
+                          <li class="language">CSS</li>
+                          <li class="language">javascript</li>
+                          </ul>
+                      </div>
+                      <div><button class="see-project" type="button">See project</button></div>
```

Comment:
- [ ] Kindly make sure your see project buttons meet the specifications of the Figma design

Your button:
![Screenshot from 2023-02-21 15-39-25](https://user-images.githubusercontent.com/97834160/220361318-cceb4404-4a2f-4340-821e-3a8e1896eb5c.png)
Figma:
![Screenshot from 2023-02-21 15-40-54](https://user-images.githubusercontent.com/97834160/220361509-14f02687-800d-445f-ac5b-326b3373fff1.png)